exercises/perfect-numbers/example.rs

```diff
@@ -3,12 +3,10 @@ pub fn classify(num: u64) -> Option<Classification> {
         return None;
     }
     let sum: u64 = (1..num).filter(|i| num % i == 0).sum();
-    if sum == num {
-        Some(Classification::Perfect)
-    } else if sum < num {
-        Some(Classification::Deficient)
-    } else {
-        Some(Classification::Abundant)
+    match sum.cmp(&num) {
```

Comment:
I notice that putting this whole thing in a `Some` is possible, but right now we are just rewriting `if`s, not doing that.